ce/src/test/scala/lgbt/princess/lifts/instances/CEImplicitSummoningTests.scala

```diff
@@ -0,0 +1,51 @@
+package lgbt.princess.lifts
+package instances
+
+import cats.data.OptionT
+import cats.effect.{IO, LiftIO}
+import cats.effect.kernel.{MonadCancel, Resource}
+import lgbt.princess.lifts.instances.ce._
+
+object CEImplicitSummoningTests {
+
+  def `Resource instances for IO`(): Unit = {
+    LiftValue[IO, Resource[IO, *]]
+    LiftScope[IO, Resource[IO, *]]
+    LiftKind[IO, Resource[IO, *]]
+    MapK[IO, OptionT[IO, *], Resource[IO, *], Resource[OptionT[IO, *], *]]
+    MapK.Derived[IO, OptionT[IO, *], Resource]
+    LiftScopeAlt[IO, Resource[IO, *]]
+    LiftKind1[IO, Resource[IO, *]]
+    LiftKind2[IO, OptionT[IO, *], Resource[IO, *], Resource[OptionT[IO, *], *]]
+    LiftKind2.Derived[IO, OptionT[IO, *], Resource]
+  }
+
+  def `Resource instances for abstract types`[F[_], G[_]]()(implicit
+      F: MonadCancel[F, ?],
+      G: MonadCancel[G, ?]
+  ): Unit = {
+    LiftValue[F, Resource[F, *]]
+    LiftScope[F, Resource[F, *]]
+    LiftKind[F, Resource[F, *]]
+    MapK[F, G, Resource[F, *], Resource[G, *]]
+    MapK.Derived[F, G, Resource]
+    LiftScopeAlt[F, Resource[F, *]]
```

Comment:
the summons for `LiftValue`, `LiftScope`, `MapK` and `LiftScopeAlt` all have ambiguous implicits, and I'm not sure why (clearly I'm doing the `LowPriorityImplicits` thing wrong, but I don't know in what way)